spacy_loggers/util.py

```diff
@@ -49,3 +52,34 @@ def dict_to_dot(obj: Dict[str, dict]) -> Dict[str, Any]:
     RETURNS (Dict[str, Any]): The key/value pairs.
     """
     return {".".join(key): value for key, value in walk_dict(obj)}
+
+
+def setup_custom_stats_matcher(
```

Comment:
The name of this function can probably be much more generic, as it really just takes a list of regular expressions and provides a function to match them against text

Reply:
I've renamed it to `matcher_for_regex_patterns`.